scss/_type.scss

```diff
@@ -1,27 +1,24 @@
-a {
-  &.tooltip-link {
-    > .material-icons {
-      color: $medium-gray;
-      vertical-align: middle;
+.tooltip-link {
+  > .material-icons {
+    color: $medium-gray;
+    vertical-align: middle;
 
-      &:hover {
-        color: theme-color('primary');
-      }
+    &:hover {
+      color: theme-color("primary");
```

Comment:
That's the hover effect I mentioned earlier which would be nice for `button` tags

Reply:
We'll think about it in the button improvement issue or another issue